src/components/breadcrumbs/_macro.spec.js

```diff
@@ -97,23 +97,23 @@ describe('macro: breadcrumbs', () => {
   it('has provided `url` on `link` elements', () => {
     const $ = cheerio.load(renderComponent('breadcrumbs', EXAMPLE_BREADCRUMBS));
 
-    const urls = mapAll($('.ons-breadcrumbs__link'), node => node.attr('href'));
-    expect(urls).toEqual(['https://example.com/', 'https://example.com/guide/']);
```

Comment:
This test should be left as it is. This is testing that the urls being passed in on line 30 and 39 are being set correctly